ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Operation/04_CreateManualBooking.md

[!!User Interface Postings](../UserInterface/01a_Bookings.md)  
[!!User Interface Select fiscal year](../UserInterface/00a_FiscalYear.md)   
[!!User Interface FY/MONTH/JOURNAL](../UserInterface/00_Book.md#fymonthjournal)  

# Create a manual posting

> Even though transactions are usually posted automatically through the automated functions across the different modules, you can also record postings manually. In some cases, it is necessary to record transactions manually when a transaction has been performed but not yet recorded, for example a cash payment or an issued invoice.

Reviewer: ```suggestion
Even though a lot of transactions can be posted automatically through the automated functions across the different modules, manual record postings are still necessary.
```

#### Prerequisites

A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).

#### Procedure

*Accounting > Post > Tab POSTINGS*

![Postings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/Bookings.png "[Postings]")

1. If necessary, click the *Currency code* drop-down list and select the appropriate currency for the posting. All available currencies are displayed in the list. By default, the base currency is preselected.

2. Enter the posting amount in the *Turnover* field. This field is mandatory.

   > [Info] The fields *Turnover*, *Contra account*, *Date* and *Account* must be filled to create a manual posting.

3. Enter an account number in the *Contra account* field.

  > [Info] After you have entered at least two numbers in the field, a list is displayed below the field with all accounts matching the entry. You can also select the corresponding account in the list.

4. Enter a reference number, usually the invoice number, in the *Receipt 1* field.

>  > [Info] Even though it is not mandatory, it is highly recommended to fill in the *Receipt 1* field, as it is used as a reference to identify and assign any subsequent postings, for example for an invoice settlement or a credit note in case of refund.

Reviewer: ```suggestion
_Satz löschen! GOBD verlangen Belegnummer!_
```

5. Enter the date for the manual posting in the *Date* field. The valid date format is *DDMMYY*.

6. Enter an account number in the *Account* field.

  > [Info] After you have entered at least two numbers in the field, a list is displayed below the field with all accounts matching the entry. You can also select the corresponding account in the list.

> 7.  If desired, enter any other relevant manual posting information in the corresponding fields.

Reviewer: ```suggestion
7.  If desired, enter any other relevant manual posting information in the corresponding fields, for example the detailed description of the transaction background or the cost unit / object number in case you work with cost accounting.
```

[!!User Interface bookings](../UserInterface/01a_Bookings.md)  
[!!User Interface select fiscal year](../UserInterface/00a_FiscalYear.md)   
[!!User Interface FY/MONTH/JOURNAL](../UserInterface/00_Book.md#fymonthjournal)  

# Create a manual booking

> Even though the system usually books transactions automatically through the automated functions across the different modules, the *Accounting* module also allows to record bookings manually. In some cases, it may be necessary to record transactions manually when a transaction has been performed but not yet recorded, for example a cash payment or an issued invoice.

Reviewer 2: ```suggestion
Even though transactions are usually posted automatically through the automated functions across the different modules, you can also record posting manually. In some cases, it is necessary to record transactions manually when a transaction has been performed but not yet recorded, for example a cash payment or an issued invoice.
```

>  > [Info] It is essential to select the correct booking period, that is, the correct fiscal year, month and journal, before making a manual booking. For detailed information, see [Select fiscal year](./01_SelectFiscalYear).

Reviewer 2: Ist booking period nicht nur der ausgewählte Monat? (Spaltenname BP für ausgewählten Monat)

#### Prerequisites

> - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
> - A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).
> - A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).

Reviewer 2: ```suggestion
A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
```

#### Procedure

*Accounting > Select fiscal year > Book > Tab BOOKINGS*

![Bookings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/Bookings.png "[Bookings]")

> 1. Select the appropriate currency for the booking in the *Currency code* drop-down list.

Reviewer 2: ```suggestion
1. Click the *Currency code* drop-down list and select the appropriate currency for the booking. All available currencies are available.
```

>   > [Info] The booking numbers must be sequential and are added by the system automatically.

Reviewer 2: Wozu gehört das? Braucht es diese Info?

> 2. Enter a value in the *Turnover* field.

Reviewer 2: ```suggestion
2. Enter the posting amount in the *Turnover* field.
```

3. Enter an account number in the *Contra account* field.

>   > [Info] When you start typing in an account number, a context menu is displayed. You can also select the appropriate option from the menu.

Reviewer 2: ```suggestion
  > [Info] After you have entered at least two numbers in the field, a list below the field displays all accounts that match the entry. You can also select the corresponding account in the list.
```

> 4. Enter the date for the manual booking in the *Date* field. Date format must be DDMMYY.

Reviewer 2: ```suggestion
4. Enter the date for the manual booking in the *Date* field. The valid date format is *DDMMYY*.
```

5. Enter an account number in the *Account* field.

> 

Reviewer 2: ```suggestion
  > [Info] After you have entered at least two numbers in the field, a list is displayed below the field with all accounts matching the entry. You can also select the corresponding account in the list.
```

>   > [Info] Note that the fields *Contra account*, *Date* and *Account* must be filled out to be able to create a booking.

Reviewer 2: ```suggestion
  > [Info] Note that the fields *Contra account*, *Date* and *Account* must be filled out to create a manual booking.
```
Die Info würde ich ganz an den Anfang stellen, also bevor diese Felder ausgefüllt werden - muss nicht auch das Feld Turnover ausgefüllt werden?

> 6. Enter any other relevant booking information in the corresponding fields.

Reviewer 2: ```suggestion
6.  If desired, enter any other relevant manual booking information in the corresponding fields.
```

>   > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent bookings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.

Reviewer 2: ```suggestion
  > [Info] Even though it is not mandatory, it is highly recommended to fill in the *Receipt 1* field, as it is used as a reference to identify and assign any subsequent postings, for example for an invoice settlement or a credit note in case of refund.
```

7. Click the [BOOK] button.  

> A message above the input line confirms that the booking is recorded. The newly created manual booking is displayed in the bookings list.

Reviewer 2: ```suggestion
The manual posting is created. The notice "Posted." is displayed above the input line. The manual posting is displayed in the postings list.
```
